onlineweb4/context_processors.py

```diff
@@ -1,8 +1,42 @@
+# -*- coding: utf-8 -*-
+
 from django.conf import settings
+from django.utils import timezone
+
+from apps.feedback.models import FeedbackRelation
 
 
 def analytics(request):
-	context_extras = {}
-	if hasattr(settings, 'GOOGLE_ANALYTICS_KEY'):
-		context_extras['GOOGLE_ANALYTICS_KEY'] = settings.GOOGLE_ANALYTICS_KEY
-	return context_extras
+    context_extras = {}
+    if hasattr(settings, 'GOOGLE_ANALYTICS_KEY'):
+        context_extras['GOOGLE_ANALYTICS_KEY'] = settings.GOOGLE_ANALYTICS_KEY
+    return context_extras
+
+
+def feedback_notifier(request):
+    context_extras = {}
+    context_extras['feedback_pending'] = []
+    if not request.user.is_authenticated():
+        return context_extras
+
+    active_feedbacks = FeedbackRelation.objects.filter(active=True)
+    for active_feedback in active_feedbacks:
+        if active_feedback.content_object is None:
+            continue
+
+        # Making sure we have an end data, and that the event is over
+        # and that the feedback deadline is not passed (logic reused from apps.feedback.mommy)
```

Comment:
Is this copy pasta from feedback mommy? If so, could you make it a more generic function and call that one instead? So we don't have the same code two places and forget to update one of them in the future..